prettyjson/prettyjson.d.ts

```diff
@@ -0,0 +1,55 @@
+// Type definitions for prettyjson
+// Project: https://github.com/rafeca/prettyjson
+// Definitions by: Wael BEN ZID EL GUEBSI <https://github.com/benzid-wael/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+
+declare module prettyjson {
+
+  /**
+   * Defines prettyjson version
+   */
+  export var version: string;
+
+  /**
+   * Render pretty json.
+   *
+   * @param data {any} Data to prettify.
+   * @param options {IOptions} Hash with different options to configure the renderer.
+   * @param indentation {number} Indentation size.
+   *
+   * @return {string} pretty serialized json data ready to display.
+     */
+  export function render(data: any, options?: IOptions, indentation?: number): string;
+
+  /**
+   * Render pretty json from a string.
+   *
+   * @param data {string} Serialized JSON data to prettify.
+   * @param options {IOptions} Hash with different options to configure the renderer.
+   * @param indentation {number} Indentation size.
+   *
+   * @return {string} pretty serialized json data ready to display.
+     */
+  export function renderString(data: string, options?: IOptions, indentation?: number): string;
+
+  export interface IOptions {
```

Comment:
http://www.typescriptlang.org/Handbook#writing-dts-files

> Naming Conventions
In general, do not prefix interfaces with I (e.g. IColor). Because the concept of an interface in TypeScript is much more broad than in C# or Java, the IFoo naming convention is not broadly useful.